docker-compose.yaml

```diff
@@ -0,0 +1,42 @@
+services:
+  postgres:
+    image: postgres:16
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
+    healthcheck:
+      test: pg_isready -d db_prod
+      interval: 10s
+      timeout: 3s
+      retries: 3
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    ports:
+      - 5432:5432
+
+  localstack:
```

Comment:
I can't see this hostname referenced anywhere, so I'm not sure this is being used yet. However, we don't need it for now as no one is currently developing it, so I'm fine to figure that out later.